### PR TITLE
Initial updates to add HPE Primera and Pure Flasharray documentation

### PR DESCRIPTION
**Pull request by rg9975**
Providing documentation for configuration of HPE Primera and Pure Flasharray plugins targeted for 4.19.

----
📚 Documentation preview 📚: https://cloudstack-documentation--363.org.readthedocs.build/en/363/

### CODE REVIEW

**Review by shwstppr — APPROVED** [review]
Not sure about the additional whitespace changes but looks okay to me

File: source/installguide/configuration.rst [hunk 1]
```diff
@@ -512,7 +512,7 @@ Basic Zone Configuration
    -  **Name.** The name of the storage device.
 
    -  **Protocol.** For XenServer, choose either NFS, iSCSI, or
-      PreSetup. For KVM, choose NFS, SharedMountPoint,CLVM, RBD or custom (for PowerFlex). For
+      PreSetup. For KVM, choose NFS, SharedMountPoint,CLVM, RBD, FiberChannel or custom (for PowerFlex). For
```
Comment (andrijapanicsb) [inline source/installguide/configuration.rst:515]: The naming is not ideal, as fibrechannel can also (has been historically) configured outside of ACS, manuially, and just a Shared Mount Point would be feed to ACS. But leave it as it is.

**Comment by andrijapanicsb** [comment]
white spaces probably removed automatically, for whatever reasons, nothing broken.

I have not reviewed the HPE-specific content, as I have no knowledge on it - so I assume this is OK.

**Review by andrijapanicsb — APPROVED** [review]

File: source/installguide/configuration.rst [hunk 2]
```diff
@@ -1460,6 +1460,121 @@ storpool.volume.tags.checkup      Minimal interval (in seconds) to check and rep
 storpool.snapshot.tags.checkup    Minimal interval (in seconds) to check and report if a StorPool Snapshot created by CloudStack exists in CloudStack's database
 ================================= ====================================================================================================================================================================
 
+HPE Primera/3PAR Plug-in
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This plugin enables Hewlett Packard Enterprise (HPE) Primera (previously 3PAR) storage systems with FiberChannel on KVM hypervisors.
+
+This documentation assumes you have the following configured in your environment before configuring a storage pool in cloudstack:
```
Comment (rajujith) [inline source/installguide/configuration.rst:1468]: The formatting is not proper. Same for Pure.
Screenshot 2023-12-11 at 10 19 24 AM

**Review by rajujith — CHANGES_REQUESTED** [review]

**Comment by andrijapanicsb** [comment]
@rg9975 do you mind pls fixing these, so we can merge it?

**Comment by rajujith** [comment]
@blueorangutan docbuild

**Comment by rajujith** [comment]
@rg9975  the bullet points are still not formatted properly.

**Comment by shwstppr** [comment]
@rajujith can you please check now?

**Comment by rajujith** [comment]
@blueorangutan docbuild

**Review by rajujith — APPROVED** [review]
LGTM

**Pull request merged**